code/modules/admin/admin_verbs.dm

```diff
@@ -79,6 +79,12 @@ GLOBAL_PROTECT(admin_verbs_admin)
 	/datum/admins/proc/paintings_manager,
 	/datum/admins/proc/display_tags,
 	/datum/admins/proc/fishing_calculator,
+	/client/proc/list_bombers,
```

Comment:
why are these all separate verbs, just make one that pulls up a pick list of all of these

Reply:
yeah, do this
It'd be like the 'Investigate' verb, which is more compact.
Additionally you could just integrate it into the Investigate verb

Reply:
I want to be able to type in `list-signallers` or `list-bombers` or even `list-law-changes` rather than have yet ANOTHER pick list (the Secrets menu already pretty much serves this function) rather than have to do some sort of advanced proc call to get to the information if secrets panel isn't working.

Reply:
i agree with san